modules/system/assets/ui/less/loader.less

```diff
@@ -12,6 +12,10 @@
 @color-stripe-loader:                        @brand-accent;
 @loader-image-path:                          @image-path;
 
+.form-buttons .loading-indicator {
```

Comment:
Minor quibble, but please put more specific rules lower in the file. So in this case put this below the `.loading-indicator` rule group

Reply:
@LukeTowers done :+1: